docker-compose.yml

```diff
@@ -0,0 +1,32 @@
+services:
+  integration_tests:
+    build:
+      dockerfile: Dockerfile
+      context: ${GSY_E_TARGET_REPO:-./}
+    container_name: ${CONTAINER_NAME:-gsy-e-tests}
+    entrypoint: ["gsy-e"]
+    command: ${COMMAND:-}
+    environment:
+      - REDIS_URL=${REDIS_URL:-redis://redis.container:6379/}
+    volumes:
+      - ${GSY_E_TARGET_REPO:-integration_tests}:/gsy-e
+
+  redis:
+    image: redis:6.2.5
+    restart: always
+    container_name: gsy-e-tests-redis
+    ports:
+      - "${PORT:-3679}:${PORT:-6379}"
```

Comment:
As a frequent user of redis-cli I would very much prefer to maintain the external port to 6379 so that it works for debugging even with this setup. Most of us are aware anyways that we need to stop any system-installed Redis / Postgres (does not apply for this PR of course) before running a docker-compose command that includes Redis.

Reply:
This is actually a typo. It should be set to "6379". Thanks for pointing it out!